docs/sources/tempo/setup/operator/multitenancy.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Setup multi-tenancy
+description: Set up multi-tenancy
+menuTitle: Multi-tenancy
+weight: 250
+aliases:
+- /docs/tempo/operator/multitenancy
+---
+
+# Multi-tenancy
+
+Tempo is a multi-tenant distributed tracing backend. It supports multi-tenancy through the use of a header: `X-Scope-OrgID`.
+See [multi-tenancy docs]({{< relref "../../operations/multitenancy" >}}) for more details. This document outlines how to deploy and use multi-tenant Tempo with the Operator.
+
+## Multitenancy without authentication
```

Comment:
Is it "multi-tenancy" or "multitenancy".

Perhaps @knylander-grafana can advise and we can lock this down in the Writers' Toolkit word list https://grafana.com/docs/writers-toolkit/write/style-guide/word-list/

Reply:
multi-tenancy is used in other pages

Reply:
```suggestion
## Multi-tenancy without authentication
```